CRITICAL - 2025-07-16 07:38:37 --> Error: Class "Services" not found
[Method: POST, Route: api/auth/login]
in APPPATH\Config\Routes.php on line 9.
 1 SYSTEMPATH\Router\RouteCollection.php(339): require()
 2 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 07:38:44 --> Error: Class "Services" not found
[Method: POST, Route: api/auth/login]
in APPPATH\Config\Routes.php on line 9.
 1 SYSTEMPATH\Router\RouteCollection.php(339): require()
 2 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 07:39:20 --> Error: Class "Services" not found
[Method: POST, Route: api/auth/login]
in APPPATH\Config\Routes.php on line 9.
 1 SYSTEMPATH\Router\RouteCollection.php(339): require()
 2 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 07:40:46 --> Error: Class "Services" not found
[Method: POST, Route: api/auth/login]
in APPPATH\Config\Routes.php on line 9.
 1 SYSTEMPATH\Router\RouteCollection.php(339): require()
 2 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 07:41:13 --> Error: Class "Services" not found
[Method: POST, Route: api/auth/login]
in APPPATH\Config\Routes.php on line 9.
 1 SYSTEMPATH\Router\RouteCollection.php(339): require()
 2 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-16 08:43:05 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-16 08:43:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:43:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:05\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655385_5ed0b7fb11cda75fca99.jpg\', \'WFO\', \'2025-07-16 08:43:05\', \'2025-07-16 08:43:05\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:05\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655385_5ed0b7fb11cda75fca99.jpg\', \'WFO\', \'2025-07-16 08:43:05\', \'2025-07-16 08:43:05\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:43:05 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:05\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655385_5ed0b7fb11cda75fca99.jpg\', \'WFO\', \'2025-07-16 08:43:05\', \'2025-07-16 08:43:05\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:05\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655385_5ed0b7fb11cda75fca99.jpg\', \'WFO\', \'2025-07-16 08:43:05\', \'2025-07-16 08:43:05\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:05\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655385_5ed0b7fb11cda75fca99.jpg\', \'WFO\', \'2025-07-16 08:43:05\', \'2025-07-16 08:43:05\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-16 08:43:13 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-16 08:43:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:43:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:13\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655393_141f2f25b25b24886c0a.jpg\', \'WFO\', \'2025-07-16 08:43:13\', \'2025-07-16 08:43:13\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:13\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655393_141f2f25b25b24886c0a.jpg\', \'WFO\', \'2025-07-16 08:43:13\', \'2025-07-16 08:43:13\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:43:13 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:13\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655393_141f2f25b25b24886c0a.jpg\', \'WFO\', \'2025-07-16 08:43:13\', \'2025-07-16 08:43:13\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:13\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655393_141f2f25b25b24886c0a.jpg\', \'WFO\', \'2025-07-16 08:43:13\', \'2025-07-16 08:43:13\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:13\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655393_141f2f25b25b24886c0a.jpg\', \'WFO\', \'2025-07-16 08:43:13\', \'2025-07-16 08:43:13\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-16 08:43:23 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-16 08:43:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:43:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:23\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655403_25e3c1e2fc16b1592ae5.jpg\', \'WFO\', \'2025-07-16 08:43:23\', \'2025-07-16 08:43:23\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:23\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655403_25e3c1e2fc16b1592ae5.jpg\', \'WFO\', \'2025-07-16 08:43:23\', \'2025-07-16 08:43:23\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:43:23 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:23\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655403_25e3c1e2fc16b1592ae5.jpg\', \'WFO\', \'2025-07-16 08:43:23\', \'2025-07-16 08:43:23\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:23\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655403_25e3c1e2fc16b1592ae5.jpg\', \'WFO\', \'2025-07-16 08:43:23\', \'2025-07-16 08:43:23\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:43:23\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655403_25e3c1e2fc16b1592ae5.jpg\', \'WFO\', \'2025-07-16 08:43:23\', \'2025-07-16 08:43:23\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-16 08:49:27 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-16 08:49:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:49:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:49:27\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655767_ad3fd8f9bb1fab8361c4.jpg\', \'WFO\', \'2025-07-16 08:49:27\', \'2025-07-16 08:49:27\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:49:27\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655767_ad3fd8f9bb1fab8361c4.jpg\', \'WFO\', \'2025-07-16 08:49:27\', \'2025-07-16 08:49:27\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 08:49:27 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:49:27\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655767_ad3fd8f9bb1fab8361c4.jpg\', \'WFO\', \'2025-07-16 08:49:27\', \'2025-07-16 08:49:27\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:49:27\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655767_ad3fd8f9bb1fab8361c4.jpg\', \'WFO\', \'2025-07-16 08:49:27\', \'2025-07-16 08:49:27\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'08:49:27\', \'-6.3429389,106.7892633\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+WJV, Gandul, Cinere, 16514, Indonesia\', \'1752655767_ad3fd8f9bb1fab8361c4.jpg\', \'WFO\', \'2025-07-16 08:49:27\', \'2025-07-16 08:49:27\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-16 09:35:22 --> mysqli_sql_exception: Duplicate entry '1-2025-07-16' for key 'attendances.user_id_attendance_date' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-16 09:35:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1-2025-07-16' for key 'attendances.user_id_attendance_date'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 09:35:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1-2025-07-16' for key 'attendances.user_id_attendance_date'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'09:35:22\', \'-6.3431977,106.7887796\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+MC4, Gandul, Cinere, 16514, Indonesia\', \'1752658522_357dbbccfc7797fe8a5f.jpg\', \'WFO\', \'2025-07-16 09:35:22\', \'2025-07-16 09:35:22\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'09:35:22\', \'-6.3431977,106.7887796\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+MC4, Gandul, Cinere, 16514, Indonesia\', \'1752658522_357dbbccfc7797fe8a5f.jpg\', \'WFO\', \'2025-07-16 09:35:22\', \'2025-07-16 09:35:22\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-16 09:35:22 --> [Caused by] mysqli_sql_exception: Duplicate entry '1-2025-07-16' for key 'attendances.user_id_attendance_date'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'09:35:22\', \'-6.3431977,106.7887796\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+MC4, Gandul, Cinere, 16514, Indonesia\', \'1752658522_357dbbccfc7797fe8a5f.jpg\', \'WFO\', \'2025-07-16 09:35:22\', \'2025-07-16 09:35:22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'09:35:22\', \'-6.3431977,106.7887796\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+MC4, Gandul, Cinere, 16514, Indonesia\', \'1752658522_357dbbccfc7797fe8a5f.jpg\', \'WFO\', \'2025-07-16 09:35:22\', \'2025-07-16 09:35:22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (\'1\', \'2025-07-16\', \'09:35:22\', \'-6.3431977,106.7887796\', \'Shift: Shift 2 (15:00 - 23:00), Alamat: MQ4Q+MC4, Gandul, Cinere, 16514, Indonesia\', \'1752658522_357dbbccfc7797fe8a5f.jpg\', \'WFO\', \'2025-07-16 09:35:22\', \'2025-07-16 09:35:22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `attendance_date`, `time_in`, `location_in`, `notes`, `photo_in`, `work_type`, `created_at`, `updated_at`) VALUES (:user_id:, :attendance_date:, :time_in:, :location_in:, :notes:, :photo_in:, :work_type:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(47): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
